development/Dockerfile

```diff
@@ -76,3 +76,5 @@ RUN pip install -c constraints.txt -r poetry_freeze_dev.txt
 
 COPY development/nautobot_config.py ${NAUTOBOT_ROOT}/nautobot_config.py
 # !!! USE CAUTION WHEN MODIFYING LINES ABOVE
+
+USER root
```

Comment:
Is this a standard pattern for 2.0 migrations? If so, why?

Reply:
Temporary workaround for a bug in the rc1 docker images. Will be fixed/unnecessary in rc2 and later.

Reply:
This was added temporarily to address an issue with coverage not working in CI. Will be removed in future when we have a container that doesn't have the issue.